test/protocol/MetaTransactionsHandlerTest.js

```diff
@@ -939,7 +1103,7 @@ describe("IBosonMetaTransactionsHandler", function () {
               s,
               v
             )
-          ).to.revertedWith(RevertReasons.INVALID_FUNCTION_SIGNATURE);
+          ).to.revertedWith(RevertReasons.FUNCTION_NOT_WHITELISTED);
         });
 
         context("Reentrancy guard", async function () {
```

Comment:
General comment: `isFunctionWhitelisted` is an external function, so there should be some separate test cases for it.

Reply:
I added an `isFunctionWhitelisted` context